packages/apollo-link-ws/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "apollo-link-ws",
+  "version": "0.0.0",
+  "description": "WebSocket transport layer for GraphQL",
+  "author": "Evans Hauser <evanshauser@gmail.com>",
+  "contributors": [
+    "James Baxley <james@meteor.com>",
+    "Jonas Helfer <jonas@helfer.email>",
+    "jon wong <j@jnwng.com>",
+    "Sashko Stubailo <sashko@stubailo.com>"
+  ],
+  "license": "MIT",
+  "main": "./lib/webSocketLink.js",
+  "typings": "./lib/webSocketLink.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apollographql/apollo-link.git"
+  },
+  "bugs": {
+    "url": "https://github.com/apollographql/apollo-link/issues"
+  },
+  "homepage": "https://github.com/apollographql/apollo-link#readme",
+  "scripts": {
+    "pretest": "npm run build",
+    "test": "npm run test-only --",
+    "posttest": "npm run lint",
+    "test-only": "mocha --reporter spec --full-trace dist/tests/tests.js",
+    "test-watch": "mocha --reporter spec --full-trace dist/tests/tests.js --watch",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha -- --reporter dot --full-trace dist/tests/tests.js",
+    "postcoverage": "remap-istanbul --input coverage/coverage.json --type lcovonly --output coverage/lcov.info",
+    "lint": "tslint --type-check -p tsconfig.json src/*.ts && tslint --type-check -p tsconfig.json tests/*.ts",
+    "prebuild": "npm run clean:dist",
+    "build": "tsc -p .",
+    "postbuild": "cp -R ./dist/src/. ./lib",
+    "watch": "tsc -w -p .",
+    "clean": "npm run clean:dist && npm run clean:coverage",
+    "clean:dist": "rimraf dist/* && rimraf lib/*",
+    "clean:coverage": "rimraf coverage/*",
+    "prepublishOnly": "npm run clean && npm run build"
+  },
+  "dependencies": {
+    "apollo-link-core": "^0.1.1",
+    "graphql": "^0.10.5",
```

Comment:
Sweet! We can just use `subscriptions-transport-ws` directly as a link, since it has the same signature. This is incredible!

CC @stubailo